Protocol.md

```diff
@@ -589,8 +590,8 @@ Frame Contents
                                 Metadata
 ```
 
-* __Frame Type__: (16) 0x0C
-* __Stream ID__: Must be 0 to pertain to the entire connection.
+* __Stream ID__: (32 bits = max value 2^31-1 = 2,147,483,647) Must be 0 to pertain to the entire connection.
```

Comment:
I suggest we don't mention hardcoded fields, so I would delete this line. See #167 and #174.